### PR TITLE
Clarifications in Okta SSO doc

### PR DESCRIPTION
**Pull request by jof**
Clarify the usage of the `attributes_to_roles` parameter and reorder some mapping keys for clarity in the example SAML connector resource.

### CODE REVIEW

File: docs/pages/access-controls/sso/okta.mdx [hunk 1]
```diff
@@ -116,8 +119,8 @@ $ tctl create okta-connector.yaml
 We are going to create a new role that'll pull in external information from Okta. Notice
 `{{external.username}}` login. It configures Teleport to look at *"username"* Okta claim
 and use that field as an allowed login for each user.  This example uses email as the
-username format.  The `email.local(external.trait)` function will remove the `@domain`
-and just have the username prefix.
+username format.  The `email.local(external.username)` function call will remove
+the `@domain` and just have the username prefix.
```
Comment (alexfornuto) [inline docs/pages/access-controls/sso/okta.mdx:123]:
```suggestion
the `@domain` and leave the username prefix.
```

**Review by alexfornuto — APPROVED** [review]
Nice! This answers a question I had myself earlier this week.

File: docs/pages/access-controls/sso/okta.mdx [hunk 2]
```diff
@@ -103,6 +103,9 @@ Now, create a SAML connector [resource](../../reference/resources.mdx):
 ```
 The above example includes the `entity_descriptor` field as multiple lines, although a single line with no spaces will also be accepted.
 
+The `attributes_to_roles` listing in the Spec maps key/value-like attributes of
```
Comment (ptgott) [inline docs/pages/access-controls/sso/okta.mdx:106]:
```suggestion
The `attributes_to_roles` field in the connector resource maps key/value-like attributes of
```

File: docs/pages/access-controls/sso/okta.mdx [hunk 3]
```diff
@@ -103,6 +103,9 @@ Now, create a SAML connector [resource](../../reference/resources.mdx):
...
+the assertion from Okta into a list of Teleport Roles to apply to the session.
```
Comment (ptgott) [inline docs/pages/access-controls/sso/okta.mdx:107]:
```suggestion
the assertion from Okta into a list of Teleport roles to apply to the session.
```

**Review by ptgott — APPROVED** [review]
Approved with minor suggestions

**Review by stevenGravy — APPROVED** [review]

**Review by zmb3 — APPROVED** [review]

**Pull request merged**